test/e2e/user_management_test.go

```diff
@@ -363,6 +363,32 @@ func (s *userManagementTestSuite) TestUserDeactivation() {
 			})
 		})
 	})
+
+	s.T().Run("test reset deactivating state when promoting user", func(t *testing.T) {
```

Comment:
Shouldn't we move this tests to the other promoting tests rather than keep it in the user deactivation test?

Reply:
Makes sense. Done in 8820c0ae7b8ae2b27ebcd6664371945e5ca01307